### DISCUSSION TITLE
How do I move these button a little bit higher?

### DISCUSSION THREAD

**Discussion by CaptGBread** [soulhotel/FF-ULTIMA#37] [Q&A]
These button looks like it doesn't have bottom margin? I would like to move it a little bit so it doesn't stick to the bottom if possible. I managed to change or customize several things, except this one. 
![Screenshot 2024-05-01 113807](https://github.com/soulhotel/FF-ULTIMA/assets/168618405/26d9eeca-36d1-4c79-b2ce-70b0ed6296ac)

**Comment by soulhotel** [comment]
you can do this in two ways

1. line 38 in `function-mini-button-bar.css`. Add your preferred margins to the button.

or 2. line 7 in `function-mini-button-bar.css`. Add padding to the bottom of the tabs container. 

Will be fine across different size settings. maybe I'll implement this into the next release.

(two pictures)
![2024-05-01_02-25](https://github.com/soulhotel/FF-ULTIMA/assets/155501797/d62299f1-ed90-419e-818b-8668b167952f)
![2024-05-01_02-39](https://github.com/soulhotel/FF-ULTIMA/assets/155501797/52e0269b-5603-4bd3-a7ba-e0f9763d1a2d)

**Reply by CaptGBread** [reply]
Thank you for your response. I tried the 2nd way and it worked! and thank you for making this beautiful theme.

**Reply by soulhotel** [reply]
np!
